migtests/scripts/functions.sh

```diff
@@ -187,10 +187,19 @@ run_sqlplus_as_sys() {
 
 
 run_sqlplus_as_schema_owner() {
-    db_name=$1
-    sql=$2
-    conn_string="${SOURCE_DB_USER_SCHEMA_OWNER}/${SOURCE_DB_USER_SCHEMA_OWNER_PASSWORD}@${SOURCE_DB_HOST}:${SOURCE_DB_PORT}/${db_name}"
-    echo exit | sqlplus -f "${conn_string}" @"${sql}"
+	if [ "$#" != 4 ]; then
```

Comment:
I'd suggest write a separate function `run_sqlplus` that takes db_name, schema, password, sql.  The name of the current function suggests it is "as schema owner"